oms-status/docker/cachet/.env.example

```diff
@@ -0,0 +1,45 @@
+APP_ENV=production
```

Comment:
let's move this to the general .env file instead?

Reply:
this is the PHP env, and it honestly never changes

Reply:
because we don't have to develop this stuff

all of this stuff in PR comes out-of-the-box